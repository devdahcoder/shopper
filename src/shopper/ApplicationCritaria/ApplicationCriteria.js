import React from 'react'
import ProfileImage from "../ProfileImage/ProfileImage"
import {RouterLink} from "../../components/Button"
import {
  ApplicationContainer,
  Form,
  FormDisplay,
  Label,
  Input,
  Select,
} from "../../components/Form";
import {ButtonContainer, Button} from "../../components/Button"
import styled from "styled-components"


const ApplicationCopyRight = styled.div`
  color: #1c1c1b;
  font-size: 14.083433px;
  font-weight: 400;
  line-height: 20px;
  width: 80%;
  margin-bottom: 1em;
`;

const LabelDiv = styled.div`
  width: 100%;
  border-bottom: 1px solid #c6c6c5;
  padding: 1em 0;
  // opacity: 0.5;
  font-size: 14px;
  font-weight: 400;
  margin-bottom: 1.5em;
`;

const SellerForm = styled.div`
  display: block;
`;

const SellerFormImage = styled.div`
  display: flex;
  align-items: center;
`;

const EmailContainer = styled.div`
  margin: 2em 0;
`;



const ApplicationCriteria = () => {

    const pixSize = "80px"
    return (
      <div>
        <ApplicationContainer>
          <ApplicationCopyRight>
            <p>
              Copyright will provide is wonderful serenity has taken possession
              of my entire as soul, like these sweet mornings of spring which I
              enjoy with my whole heart. I am alone standards.
            </p>

            <RouterLink copyRight>Read our Buyer guideline</RouterLink>
          </ApplicationCopyRight>

          <Form action="">
            <FormDisplay>
              <SellerForm>
                <LabelDiv>Your Account</LabelDiv>

                <SellerFormImage>
                  <ProfileImage pixSize={pixSize} />
                  <Input file type="file" name="" id="" />
                </SellerFormImage>

                <EmailContainer>
                  <Label htmlFor="">Email</Label>
                  <Input
                    className="email"
                    type="email"
                    name=""
                    id=""
                    placeholder="xyz.abc@deb.com"
                  />
                  <RouterLink signOut to="/">
                    Sign out and apply with another account
                  </RouterLink>
                </EmailContainer>
              </SellerForm>
              <SellerForm>
                <LabelDiv>How did you hear about Shopper</LabelDiv>

                <Select name="" id="">
                  <option value="newspaper">Newspaper</option>
                  <option value="advertisment">Advertisment</option>
                  <option value="magazine">Magazine</option>
                </Select>
              </SellerForm>
            </FormDisplay>
            <ButtonContainer>
              <Button formBtn goBack>
                <RouterLink goBack to="/application-criteria">
                  Go Back
                </RouterLink>
              </Button>

              <Button formBtn next>
                <RouterLink next to="/company-profile">
                  Next
                </RouterLink>
              </Button>
            </ButtonContainer>
          </Form>
        </ApplicationContainer>
      </div>
    );
}

export default ApplicationCriteria
