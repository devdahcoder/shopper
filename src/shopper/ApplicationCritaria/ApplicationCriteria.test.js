import React from "react";
import { render, cleanup } from "@testing-library/react";
import ApplicationCriteria from "./ApplicationCriteria"


afterEach(cleanup)

it("should take a snapshot", () => {
    const {asFragment} = render(<ApplicationCriteria />)
    expect(asFragment(<ApplicationCriteria />)).toMatchSnapshot()
})



