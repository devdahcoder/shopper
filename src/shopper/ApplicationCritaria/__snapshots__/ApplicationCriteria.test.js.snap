// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="application-criteria-container"
    >
      <div
        class="criteria-copyright"
      >
        <p>
          Copyright will provide is wonderful serenity has taken possession of my entire as soul, like these sweet mornings of spring which I enjoy with my whole heart. I am alone standards.
        </p>
        <a
          class="criteria-copyright guideline"
          href="http://www.google.com"
        >
          Read our Buyer guideline
        </a>
      </div>
      <form
        action=""
      >
        <div
          class="application-form-display"
        >
          <div
            class="seller-form"
          >
            <div
              class="label"
            >
              Your Account
            </div>
            <div
              class="seller-form-image"
            >
              <span
                class="profile-header-image"
                style="width: 80px; height: 80px;"
              >
                <img
                  alt=""
                  class="profile-header"
                  src="download.png"
                />
              </span>
              <input
                class="file"
                id=""
                name=""
                type="file"
              />
            </div>
            <div
              class="email-container"
            >
              <label
                class="email-label"
                for=""
              >
                Email
              </label>
              <input
                class="email"
                id=""
                name=""
                placeholder="xyz.abc@deb.com"
                type="email"
              />
              <a
                class="seller-log-out"
                href="http://www.google.com"
              >
                Sign out and apply with another account
              </a>
            </div>
          </div>
          <div
            class="seller-form"
          >
            <div
              class="label"
            >
              How did you hear about Shopper
            </div>
            <select
              class="select"
              id=""
              name=""
            >
              <option
                value="newspaper"
              >
                Newspaper
              </option>
              <option
                value="advertisment"
              >
                Advertisment
              </option>
              <option
                value="magazine"
              >
                Magazine
              </option>
            </select>
          </div>
        </div>
      </form>
      <div
        class="seller-btn-container"
      />
    </div>
  </div>
</DocumentFragment>
`;
